app/services/clients/stripe/stripe_client.js

```diff
@@ -37,8 +37,13 @@ module.exports = {
     return stripe.accounts.update(stripeAccountId, company.basicObject())
   },
 
-  createPerson: function (stripeAccountId, body) {
+  listPersons: async function (stripeAccountId) {
+    const persons = await stripe.accounts.listPersons(stripeAccountId)
+    return persons.data
```

Comment:
Why can’t this just do `stripe.accounts.listPersons(stripeAccountId)`?

Reply:
yeh it could, it returns an object of form:
{
     "object": "list",
     "data" : [ ... persons ... ]
}

I thought maybe it was clearer if we returned the actual persons list here, but if you think otherwise I can change it.

Reply:
I just thought it was a bit inconsistent that `updatePerson` returns immediately but the `listPersons` will wait for the request to Stripe to come back.

Anyway, happy to go with whatever you think is best.

Reply:
yeh that's fair, I can change it for consistency